diff --git a/node_modules/@react-native-community/cli-platform-ios/build/config/index.js b/node_modules/@react-native-community/cli-platform-ios/build/config/index.js
index 3aa1de6..8458587 100644
--- a/node_modules/@react-native-community/cli-platform-ios/build/config/index.js
+++ b/node_modules/@react-native-community/cli-platform-ios/build/config/index.js
@@ -59,18 +59,7 @@ function projectConfig(folder, userConfig) {
   };
 }
 function dependencyConfig(folder, userConfig = {}) {
-  if (userConfig === null) {
-    return null;
-  }
-  const podspecPath = (0, _findPodspec.default)(folder);
-  if (!podspecPath) {
-    return null;
-  }
-  return {
-    podspecPath,
-    configurations: userConfig.configurations || [],
-    scriptPhases: userConfig.scriptPhases || []
-  };
+  let configurations = []
 }
 const findPodfilePaths = _findAllPodfilePaths.default;
 exports.findPodfilePaths = findPodfilePaths;
